packages/file_selector/file_selector_platform_interface/lib/src/types/x_type_group/x_type_group.dart

```diff
@@ -14,7 +15,22 @@ class XTypeGroup {
     this.mimeTypes,
     this.macUTIs,
     this.webWildCards,
-  });
+  }) {
+    _verifyExtensions();
+  }
+
+  void _verifyExtensions() {
+    if (extensions == null) return;
```

Comment:
Could you use a local variable for extensions, mutate that, then re-assign at the end? That way you won't need `!` everywhere.

Reply:
Great catch. Done. PTAL